.gitlab-ci.yml

```diff
@@ -192,7 +192,9 @@ test:check-python3-formatting:
 
 .test:acceptance:
   stage: test
-  image: docker:dind
+  image: docker:26-cli
+  services:
+    - docker:26-dind
```

Comment:
Docker in docker as a service won't work because these tests do SSH into the virtual device container.

Reply:
Deleted the last commit.